### PR TITLE
refactor(apollo logging): Simplify the default log path to /opt/logs

### PR DESCRIPTION
**Pull request by klboke**
## What's the purpose of this PR

refactor(apollo logging): Simplify the default log path to `/opt/logs`

### motivation

The default log path for all Apollo services is prefixed with 100003172, which seems to have no additional purpose but hinders quick locating of log files. It's time to remove it.

### CODE REVIEW

**Review by Shoothzj — DISMISSED** [review]

**Comment by nobodyiam** [comment]
There is an issue that the gc.log would conflict if they are running in one machine.
https://github.com/apolloconfig/apollo/blob/2800a00e928333d579fe1e78791c82c7cf352562/apollo-configservice/src/main/scripts/startup.sh#L95-L110

**Comment by Shoothzj** [comment]
As I think, we can use a more reasonable name, like /opt/apollo/logs, not magic number

**Comment by klboke** [comment]
@nobodyiam 
> There is an issue that the gc.log would conflict if they are running in one machine.

Adding prefix of ServiceName to gc.log path

File: apollo-adminservice/src/main/scripts/startup.sh [hunk 1]
```diff
@@ -97,14 +97,14 @@ if [[ "$javaexe" ]]; then
     version=$(echo "$version" | awk -F. '{printf("%03d%03d",$1,$2);}')
     # now version is of format 009003 (9.3.x)
     if [ $version -ge 011000 ]; then
-        JAVA_OPTS="$JAVA_OPTS -Xlog:gc*:$LOG_DIR/gc.log:time,level,tags -Xlog:safepoint -Xlog:gc+heap=trace"
+        JAVA_OPTS="$JAVA_OPTS -Xlog:gc*:$LOG_DIR/$SERVICE_NAME-gc.log:time,level,tags -Xlog:safepoint -Xlog:gc+heap=trace"
```
Comment (nobodyiam) [inline apollo-adminservice/src/main/scripts/startup.sh:100]: Is it better to have the name as `apollo-configservice.gc.log` versus `apollo-configservice-gc.log`?
```suggestion
        JAVA_OPTS="$JAVA_OPTS -Xlog:gc*:$LOG_DIR/$SERVICE_NAME.gc.log:time,level,tags -Xlog:safepoint -Xlog:gc+heap=trace"
```

Comment (klboke) [reply apollo-adminservice/src/main/scripts/startup.sh:100]: `apollo-configservice.gc.log` looks better

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**